wai-handler-hal.cabal

```diff
@@ -28,7 +28,13 @@ extra-source-files:
   test/golden/WaiRequest.txt
 
 tested-with:
-  GHC ==8.6.5 || ==8.8.4 || ==8.10.7 || ==9.0.2 || ==9.2.2
+  GHC ==8.6.5
+   || ==8.8.4
```

Comment:
Pretty sure the nixpkgs doesn't go back to 8.6.5 any more, which is why I cut `aws-arn` back to `8.10.7`. Up to you

Reply:
We have CI testing it so may as well keep it until that goes away.